core/domain/classifier_services.py

```diff
@@ -303,9 +366,45 @@ def create_classifier_training_job(algorithm_id, interaction_id, exp_id,
     job_id = classifier_models.ClassifierTrainingJobModel.create(
         algorithm_id, interaction_id, exp_id, exp_version, training_data,
         state_name, status)
+
+    # Create a TrainingJobExplorationMappingModel instance in datastore.
+    create_job_exploration_mapping(exp_id, exp_version, state_name, job_id)
+
     return job_id
 
 
+def create_multi_classifier_training_jobs(job_dicts_list):
```

Comment:
Given the existence of this function, shouldn't we remove create_classifier_training_job() above altogether?